scripts/timer.js

```diff
@@ -0,0 +1,34 @@
+export default function listener() {
+    let timerInput = document.querySelector(".time");
+    let buttonRun = document.querySelector(".start");
+    let timerShow = document.querySelector(".timer");
+    let buttonPause = document.querySelector(".pause");
+    let pause = false;
+    let counter = 0;
+    buttonPause.addEventListener('click', () => {
+        pause = !pause;
+        if (pause) buttonPause.innerHTML = 'Стоп'; counter += 1;
+        if (!pause) counter -= 1; buttonPause.innerHTML = 'Продолжить';
+    });
+    buttonRun.addEventListener('click', function() {
+        let time = parseInt(timerInput.value);
+        counter = time;
+        let timer = setInterval(function () {
+            let seconds = counter;
+            if (counter <= 0) {
+                let sound = new Howl({
+                    src: ['alarm.mp3']
```

Comment:
вебпак уже предоставляет этот путь, как бы экспортируя его из импорта этого ресурса в сборке
import alarmSrc from "../alarm.mp3"
```suggestion
                    src: [alarmSrc]
```
И тебе не надо подбирать путь в сборке, поэтому везде пути ресурсов стоит брать из импорта

Reply:
ну и импорт ресурса звука надо перенести тоже сюда